src/components/Tool/FixEmailAddresses/FixEmailAddresses.tsx

```diff
@@ -288,7 +288,14 @@ export const FixEmailAddresses: React.FC<FixEmailAddressesProps> = ({
                     name={`${person.firstName} ${person.lastName}`}
                     key={person.id}
                     personId={person.id}
-                    emails={dataState[person.id]?.emailAddresses || []}
+                    email={
+                      dataState[person.id]?.emailAddresses.map((email) => ({
+                        ...email,
+                        isValid: false,
+                        personId: person.id,
+                        isPrimary: email.primary,
+                      })) || []
+                    }
```

Comment:
If needed, I can go over this in more detail on a call.

Reply:
Thanks Daniel, honestly I made this pull request way before I should have, I've been working through a bunch of errors and a lot of the code has changed since the last commit. I'll take a look at what is running right now and try and implement these fixes though.